src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+      class="form"
+    >
+      <label class="Search">
```

Comment:
```suggestion
      <label class="search search_big">
```
and use `.search_big` selector here:
![image](https://user-images.githubusercontent.com/86923089/205136053-12bc01b5-3f99-44aa-9760-62fbf0aea5e8.png)

https://github.com/mate-academy/layout_search-bar-airbnb/pull/2358#discussion_r1036868404 if you don't understand the mentor's comment or you disagree with smth - add a reply in GitHub or ask in Slack (fe_chat)

fix it everywhere in a such way